<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyMultiFileMapForm</class>
 <widget class="GUIFormBase" name="PropertyMultiFileMapForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Property MultiFileMap</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>191</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>Library</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyMultiFileMap.qrc">
     <normaloff>:/Qtres/Library.png</normaloff>:/Qtres/Library.png</iconset>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>381</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosX">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosY">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFileName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>FileName</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOverlapY">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOverlapX">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Overlap</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>381</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLineEdit" name="lineEditInspName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Path / Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditImgNum_Y">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditImgNum_X">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>画像重ね合わせ枚数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditProduct">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>製品名</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PropertyMultiFileMap.qrc"/>
 </resources>
 <connections/>
</ui>
